File: app/src/main/java/com/dinodevs/greatfitwatchface/theme/bin/StepsProgress.java

package com.dinodevs.greatfitwatchface.theme.bin;

import java.util.HashMap;
import java.util.Map;

public class StepsProgress {

    private Circle circle;
    private Map<String, Object> additionalProperties = new HashMap<String, Object>();

    public Circle getCircle() {
        return circle;
    }

    public void setCircle(Circle circle) {
        this.circle = circle;
    }

    public Map<String, Object> getAdditionalProperties() {
        return this.additionalProperties;
    }

    public void setAdditionalProperty(String name, Object value) {
        this.additionalProperties.put(name, value);
    }

}

File: app/src/main/java/com/dinodevs/greatfitwatchface/theme/bin/StepsProgress.kt
package com.dinodevs.greatfitwatchface.theme.bin

import java.util.*

class StepsProgress {
    var circle: Circle? = null
    private val additionalProperties: MutableMap<String, Any> = HashMap()

    fun getAdditionalProperties(): Map<String, Any> {
        return additionalProperties
    }

    fun setAdditionalProperty(name: String, value: Any) {
        additionalProperties[name] = value
    }
}